app/scripts/modules/core/src/naming/naming.service.ts

```diff
@@ -103,8 +103,7 @@ export class NamingService {
   }
 
   public getMoniker(app: string, stack: string, detail: string): IMoniker {
-    const cluster = this.getClusterName(app, stack, detail);
-    return { app, stack, detail, cluster };
+    return { app, stack, detail };
```

Comment:
With this, a frigga-style cluster name will still be displayed in the UI:

![screen shot 2017-12-08 at 3 03 41 pm](https://user-images.githubusercontent.com/59994/33788806-1bc116d2-dc29-11e7-9930-d04a767198af.png)

But the resulting `moniker` in the pipeline.json will be:
```
      "moniker": {
        "app": "spintest",
        "detail": "test",
        "stack": "asher"
      },
```